docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -362,6 +362,33 @@ When you are ready to migrate your eZ Publish XmlText content to the eZ Platform
 
 `php -d memory_limit=1536M bin/console ezxmltext:convert-to-richtext --export-dir=ezxmltext-export --export-dir-filter=notice,warning,error --concurrency 4 -v`
 
+**Custom tags**
+
+eZ Platform now supports custom tags, (including inline custom tags), and limited use of custom tag attributes.
+After migration to richtext, you'll need to adapt your custom tag config for eZ Platform and rewrite the custom tags in twig
+
+See [eZ Platform custom tag documentation](https://doc.ezplatform.com/en/latest/guide/extending/extending_online_editor/#custom-tags) for more info.
+If you in legacy as configured custom attributes in OE using [ezoe_attributes.ini](https://github.com/ezsystems/ezpublish-legacy/blob/master/extension/ezoe/settings/ezoe_attributes.ini#L33-L48), please note that not all types are supported.
```

Comment:
Hmm, something doesn't sound right to me (though I know what you're saying). We could use @ezsystems/documentation-team advice here :)